atoMEC/staticKS.py

```diff
@@ -645,6 +645,9 @@ def calc_E_kin_bound(orbs, xgrid):
         # multiply and sum over occupation numbers
         e_kin_dens = np.einsum("ijk,ijkl->l", orbs.occnums, kin_orbs)
 
+        # this is necessary because the Laplacian is not accurate at the boundary
```

Comment:
```suggestion
        # FIXME: this is necessary because the Laplacian is not accurate at the boundary
```

Reply:
I meant adding the comment in the code :). I changed the title of the issue to the original back again.

Reply:
Oops